reo/nested_inputs.py

```diff
@@ -1093,7 +1102,11 @@ def list_of_str(input):
           "max": 1.0e9,
           "default": 0.0,
           "description": "Maximum system size for which production-based incentives apply"
-        }
+        },
+         "emissions_factor_lb_CO2_per_gal": {
+          "type": "float",
+          "description": "Average carbon dioxide emissions factor"
```

Comment:
@TK-23 what is the average taken over? eg. Average annual, average hourly, or average over time step (could be 15 or 30 minutes)? Please update the description.